src/requirement_gatherer/graph.py

```
@@ -71,7 +77,10 @@ async def call_model(
 
         return {"messages": [msg]}
 
-    return call_model
+    if _req_gatherer_stub_enabled:
+        return call_model_stub
```

Ruff (0.8.2)

Undefined name `call_model_stub`

(F821)